examples/src/main/java/com/opengamma/strata/examples/finance/SwapPricingExample.java

```diff
@@ -143,49 +146,18 @@ private static void calculate(CalculationRunner runner) {
   //-----------------------------------------------------------------------  
   // create a vanilla fixed vs libor 3m swap
   private static Trade createVanillaFixedVsLibor3mSwap() {
-    NotionalSchedule notional = NotionalSchedule.of(Currency.USD, 100_000_000);
-
-    SwapLeg payLeg = RateCalculationSwapLeg.builder()
-        .payReceive(PayReceive.PAY)
-        .accrualSchedule(PeriodicSchedule.builder()
-            .startDate(LocalDate.of(2014, 9, 12))
-            .endDate(LocalDate.of(2021, 9, 12))
-            .frequency(Frequency.P6M)
-            .businessDayAdjustment(BusinessDayAdjustment.of(MODIFIED_FOLLOWING, HolidayCalendarIds.USNY))
-            .build())
-        .paymentSchedule(PaymentSchedule.builder()
-            .paymentFrequency(Frequency.P6M)
-            .paymentDateOffset(DaysAdjustment.NONE)
-            .build())
-        .notionalSchedule(notional)
-        .calculation(FixedRateCalculation.of(0.015, DayCounts.THIRTY_U_360))
-        .build();
-
-    SwapLeg receiveLeg = RateCalculationSwapLeg.builder()
-        .payReceive(PayReceive.RECEIVE)
-        .accrualSchedule(PeriodicSchedule.builder()
-            .startDate(LocalDate.of(2014, 9, 12))
-            .endDate(LocalDate.of(2021, 9, 12))
-            .frequency(Frequency.P3M)
-            .businessDayAdjustment(BusinessDayAdjustment.of(MODIFIED_FOLLOWING, HolidayCalendarIds.USNY))
-            .build())
-        .paymentSchedule(PaymentSchedule.builder()
-            .paymentFrequency(Frequency.P3M)
-            .paymentDateOffset(DaysAdjustment.NONE)
-            .build())
-        .notionalSchedule(notional)
-        .calculation(IborRateCalculation.of(IborIndices.USD_LIBOR_3M))
-        .build();
-
-    return SwapTrade.builder()
-        .product(Swap.of(payLeg, receiveLeg))
-        .info(TradeInfo.builder()
+    return FixedIborSwapConventions.USD_FIXED_6M_LIBOR_3M.toTrade(
```

Comment:
Suggest using a local variable for tradeInfo. The other parameters should have an end of line comment to explain what they are.